### PR TITLE
Add information_schema.foreign_table_options

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: docs/general/information-schema.rst [hunk 1]
```diff
@@ -952,6 +953,36 @@ See :ref:`administration-fdw`.
...
+
+.. list-table::
+   :header-rows: 1
+
+   * - Column Name
+     - Return Type
+     - Description
+   * - ``foreign_table_catalog``
+     - ``TEXT``
+     - Name of the database that contains the foreign table. Always ``crate``.
```
Comment (matriv) [inline docs/general/information-schema.rst:972]:
```suggestion
     - Name of the database that contains the foreign table. Value is always ``crate``.
```

File: docs/general/information-schema.rst [hunk 2]
```diff
@@ -952,6 +953,36 @@ See :ref:`administration-fdw`.
      - ``TEXT``
      - Name of the foreign server
 
+.. _foreign_table_options:
+
+``foreign_table_options``
+-------------------------
+
+Lists options for foreign tables created using :ref:`ref-create-foreign-table`.
+See :ref:`administration-fdw`.
```
Comment (matriv) [inline docs/general/information-schema.rst:962]:
```suggestion
See :ref:`administration-fdw` for more details.
```

File: server/src/test/java/io/crate/integrationtests/ForeignDataWrapperITest.java [hunk 3]
```diff
@@ -87,6 +87,20 @@ public void test_full_fdw_flow() throws Exception {
         assertThat(response).hasRows(
             "doc| dummy"
         );
+        execute(
+            """
+            SELECT
+                option_name,
```
Comment (matriv) [inline server/src/test/java/io/crate/integrationtests/ForeignDataWrapperITest.java:93]: why not selecting all columns?

File: server/src/main/java/io/crate/metadata/information/ForeignTableOptionsTableInfo.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    public static final RelationName IDENT = new RelationName(InformationSchemaInfo.NAME, NAME);
+
+    public static SystemTable<ForeignTable.Option> create() {
+        return SystemTable.<ForeignTable.Option>builder(IDENT)
+            .add("foreign_table_catalog", DataTypes.STRING, ignored -> Constants.DB_NAME)
+            .add("foreign_table_schema", DataTypes.STRING, x -> x.relationName().schema())
+            .add("foreign_table_name", DataTypes.STRING, x -> x.relationName().name())
+            .add("option_name", DataTypes.STRING, ForeignTable.Option::name)
+            .add("option_value", DataTypes.STRING, ForeignTable.Option::value)
+            .build();
```
Comment (matriv) [inline server/src/main/java/io/crate/metadata/information/ForeignTableOptionsTableInfo.java:42]: shouldn't we have the 1st 3 columns as pk?

**Review by matriv — COMMENTED** [review]
Left some comments, thx.

File: server/src/main/java/io/crate/metadata/information/ForeignTableOptionsTableInfo.java (hunk 4 above)
Comment (mfussenegger) [reply server/src/main/java/io/crate/metadata/information/ForeignTableOptionsTableInfo.java:42]: No? Usually they're referred to as view - which wouldn't have PKs

File: docs/general/information-schema.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/general/information-schema.rst:972]: I'd rather leave it for consistency with the others.

File: docs/general/information-schema.rst (hunk 2 above)
Comment (mfussenegger) [reply docs/general/information-schema.rst:962]: Same here

File: server/src/test/java/io/crate/integrationtests/ForeignDataWrapperITest.java (hunk 3 above)
Comment (mfussenegger) [reply server/src/test/java/io/crate/integrationtests/ForeignDataWrapperITest.java:95]: Added foreign_table_schema and foreign_table_name.

**Review by matriv — APPROVED** [review]
thx!

**Pull request merged**